sub-packages/bionemo-scdl/src/bionemo/scdl/index/row_feature_index.py

```diff
@@ -60,36 +63,40 @@ def __len__(self) -> int:
         """The length is the number of rows or RowFeatureIndex length."""
         return len(self._feature_arr)
 
-    def append_features(self, n_obs: int, features: pd.DataFrame, label: Optional[str] = None) -> None:
+    def append_features(self, n_obs: int, features: dict, num_genes: int, label: Optional[str] = None) -> None:
```

Comment:
you could specify a dictionary of what here, something like `dict[str, int]` maybe or whatever it really is.

Reply:
if you aren't familiar with that syntax, it's read like `dict[key_type, value_type]`